tensorboard/components/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -139,10 +144,15 @@ export const DataLoaderBehavior = {
     this.cancelAsync(this._loadDataAsync);
   },
 
+  _loadDataIfActive() {
+    if (this.active) {
+      this._loadData();
+    }
+  },
+
   _loadDataImpl() {
+    if (!this.active) return;
```

Comment:
It doesn’t look like `this.active` is ever set to `true`. What’s
happening here? How does this work?

Reply:
It is only set by parent (technically component that uses the behavior can hard code it too). 
This should gets used with `tf-category-paginted-view`. i.e.,
https://github.com/stephanwlee/tensorboard/blob/225748afd496f62387682f27b304543282bded37/tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html#L131-L138

Reply:
Right, of course. Thanks.